numpy/core/src/umath/reduction.c

```diff
@@ -166,7 +166,7 @@ PyArray_CopyInitialReduceValues(
  * identity    : If Py_None, PyArray_CopyInitialReduceValues is used, otherwise
  *               this value is used to initialize the result to
  *               the reduction's unit.
- * loop        : The loop which does the reduction.
+ * loop        : `reduce_loop` from `ufunc_object.c`.  TODO: Refactor
```

Comment:
The fact that we are passing in a function (and only ever that) from `ufunc_object.c` is a bit ridiculous.  There was probably some reason behind it.  But the only reason I didn't change it, was that it didn't seem much in the way.